### DISCUSSION TITLE
Which applications need lots of new windows?

### DISCUSSION THREAD

**Discussion by Cylocation** [paperwm/PaperWM#757] [Q&A] [answered]
Hi, I'm just curious what the use cases are for
`Super Return` or `Super N` Create a new window from the active application
I haven't found it to be useful for myself and wonder why such a valuable key combo is bound by default to this in particular. I know the original designer stopped participating in development and that the true answer might lie with them, but perhaps I'm just missing something obvious. Do people use windows instead of tabs for their browser since the navigation with windows is now preferable? Do people open lots of terminal windows? Is this just a common convention in tiling window managers?
Thanks for your insight!

**Comment by Lythenas** [comment]
I don't use this keybinding at all. I have seen in other tiling window managers where by default `Super + Return` is bound to open a new terminal.

To be honest I also don't really see much of a use case for a keybinding to open a new window for the active application. But I guess there are use cases. I think the reason we haven't removed this is that we don't want to break existing users. But not sure if anyone depends on it.

You can always just rebind the keybinding to something more useful yourself.

**Comment by paradajz** [comment, marked as answer]
I use it all the time for new terminal and new file manager window.

**Comment by Thesola10** [comment]
While I don't use this keybind for this purpose, I do tend to open a few terminal windows (at least three) on any given session, and it would also make sense for a web browser, depending on usage

**Comment by jtaala** [comment]
I also don't use that shortcut.  I have do have gnome custom shortcuts to open terminal, browser, etc. which I generally use instead.

It's a good point though - it takes two keybinds that are fairly commonly used.  If we were going to take one of them (either super+enter or super+n) - what default action would/could we bind instead?
